AudioPlayer/AudioPlayer/utils/BackgroundHandler.swift

```diff
@@ -22,7 +22,7 @@
         /// - Returns: A unique identifier for the new background task. You must pass this value to the
         ///     `endBackgroundTask:` method to mark the end of this task. This method returns `UIBackgroundTaskInvalid`
         ///     if running in the background is not possible.
-        func beginBackgroundTask(expirationHandler handler: (() -> Void)?) -> UIBackgroundTaskIdentifier
+        func beginBgTask(expirationHandler handler: (() -> Void)?) -> UIBackgroundTaskIdentifier
```

Comment:
Could you keep the `background` instead of using an abbreviation?

Reply:
It conflicts with the UIKit API, I got this warning `Function call causes an infinite recursion`

<img width="1360" alt="Screenshot" src="https://github.com/user-attachments/assets/51db0aa2-a550-4f46-9fe4-08dba01c33c1">

How about I keep it short as `beginTask` and `endTask`, since you already name the extension `BackgroundTaskCreator`

Reply:
Right, this is because it's already implemented so technically you wouldn't need to provide an implementation for it. Maybe the signature changed and that's what would need to get updated in the protocol definition

Reply:
Got what you mean. I changed it to match with UIKit definition.